pydexec/command.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+
+import os
+from subprocess import CalledProcessError, Popen
+
+from pydexec.user import User
+
+
+class Command(object):
+    def __init__(self, program):
+        self._program = program
+        self._args = []
+        self._user = None
+        self._env = dict(os.environ)
+
+    def args(self, *args):
+        """ Add a list of extra arguments to the command. """
+        self._args += list(args)
+        return self
+
+    def user(self, user):
+        """
+        Set the user to change to before execution. The ``user`` argument
+        takes the same form as that provided to the ``USER`` command in a
+        Dockerfile, so [<user>][:<group>], where both user and group are
+        optional and user can be a username or UID and group can be a group
+        name or GID.
+        """
+        self._user = User.from_spec(user)
+        return self
```

Comment:
Yes, I think it's reasonable to keep it simple like this. In the future there are going to be other things which can be set multiple times like environment variables.